__tests__/app.test.js

```diff
@@ -133,3 +133,25 @@ describe("5. GET /api/articles returns 200", () => {
       });
   });
 });
+
+describe("6. GET /api/articles/:article_id/comments returns 200", () => {
+    describe("/api/articles/:article_id/comments", () => {
+      describe("GET /api/articles/1/comments returns 200", () => {
+        test("status code: 200", () => {
+          return request(app).get("/api/articles/1/comments").expect(200);
+        });
+      });
+  
+      test("Will return array of objects, sorted by most recent comments first", () => {
+        return request(app)
+          .get("/api/articles/1/comments")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body).toBeSortedBy('created_at', {
+                descending: true,
+                coerce: true,
+            })
+        });
```

Comment:
There are a some tests missing here, please include the following tests for:

1. Status 400: Bad request
2. Status 404: Not found
3. What happens if there is a valid article_id but no comments? 👀 what test could you include for this?